contracts/src/v0.8/functions/v1_1_0/FunctionsBilling.sol

```diff
@@ -50,7 +50,8 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
     uint40 minimumEstimateGasPriceWei; //            ║ The lowest amount of wei that will be used as the tx.gasprice when estimating the cost to fulfill the request
     uint16 maxSupportedRequestDataVersion; // ═══════╝ The highest support request data version supported by the node. All lower versions should also be supported.
     uint224 fallbackNativePerUnitLink; // ═══════════╗ Fallback NATIVE CURRENCY / LINK conversion rate if the data feed is stale
-    uint32 requestTimeoutSeconds; // ════════════════╝ How many seconds it takes before we consider a request to be timed out
+    uint32 requestTimeoutSeconds; //                 ║ How many seconds it takes before we consider a request to be timed out
+    uint24 transmitTxSizeBytes; // ══════════════════╝ The size of the transmit transaction in bytes used to calculate the L1 data fee on L2s
```

Comment:
To be more specific: this is the _expected maximum_ size (in bytes) to fulfill one request. Used to estimate the cost to post from L2 to L1.